airflow/contrib/operators/gcp_vision_operator.py

```diff
@@ -206,3 +207,19 @@ def __init__(self, *args, **kwargs):
             DeprecationWarning, stacklevel=3
         )
         super().__init__(*args, **kwargs)
+
+
+class CloudVisionReferenceImageDeleteOperator(CloudVisionDeleteReferenceImageOperator):
```

Comment:
We don't need it because this operator was not released in Airflow 1.10

Reply:
Removed, thanks